Bindings/Java/tests/TestEditMarkers.java

```diff
@@ -0,0 +1,64 @@
+import org.opensim.modeling.*;
+import java.io.IOException;
+import static java.nio.file.StandardOpenOption.*;
+import java.nio.file.*;
+
+class TestEditMarkers {
+  public static void main(String[] args) {
+    try {
+        Model model = new Model("gait10dof18musc_subject01.osim");
+        OpenSimContext context=null;
+        long t1  = System.currentTimeMillis();
+        context = new OpenSimContext(model.initSystem(), model);
+        MarkerSet markerset = model.getMarkerSet();
+        // The following code exercises the Marker->Add New  functionality
+        Vec3 offset = new Vec3(0.11, 0.22, 0.33);
+        Body body = model.getBodySet().get(0);
+        String newMarkerName = "newMarker";
+        // This exercises Marker -> Add New
```

Comment:
Do you mean the following lines 19-29 are what `Add New Marker` in the GUI does?